src/libraries/Microsoft.Extensions.Configuration.Binder/tests/Common/ConfigurationBinderTests.cs

```diff
@@ -19,7 +19,7 @@ namespace Microsoft.Extensions
 {
     public partial class ConfigurationBinderTests
     {
-        [ConditionalFact(typeof(TestHelpers), nameof(TestHelpers.NotSourceGenMode))] // Need support for records.
```

Comment:
Given the feature, I think it would make sense to also include test coverage for 

1. Positional records and
2. Classes with primary constructors.

Reply:
Done.